src/frameworks/ez.md

```diff
@@ -1,21 +1,146 @@
 
 In particular, see:
  
- * The [.platform.app.yaml](https://github.com/ezsystems/ezplatform/blob/master/.platform.app.yaml) file, which automatically builds eZ Platform in dev mode or production mode depending on your defined project-level variables.
+ * The [.platform.app.yaml](https://github.com/ezsystems/ezplatform/blob/master/.platform.app.yaml) file, which
+ automatically builds eZ Platform in dev mode or production mode depending on your defined project-level variables.
  * The [.platform](https://github.com/ezsystems/ezplatform/tree/master/.platform) directory
- * The [platformsh.php](https://github.com/ezsystems/ezplatform/blob/master/app/config/env/platformsh.php) configuration file, which does the work of mapping Platform.sh environment variables into eZ Platform.  It also will automatically enable Redis-based cache and session support if detected.
+ * The [platformsh.php](https://github.com/ezsystems/ezplatform/blob/master/app/config/env/platformsh.php) configuration
+  file, which does the work of mapping Platform.sh environment variables into eZ Platform.  It also will automatically 
+  enable Redis-based cache and session support if detected.
+
+
+## Local Development with eZ Platform 2.x and later
+
+eZ systems provide a tool called [eZ Launchpad](https://ezsystems.github.io/launchpad/) for local development that
+improves Developer eXperience and reduces the container management. eZ Launchpad is ready to work with Platform.sh.
+
+### eZ Launchpad installation
+
+eZ Launchpad approach is to stay as decoupled as possible from your development machine and your remote hosting whether
+you are Linux or Mac OSX. Installation is really simple:
+
+```bash
+curl -LSs https://ezsystems.github.io/launchpad/install_curl.bash | bash
+```
+ 
+Then you can just start to use it to initialize your eZ Platform project on top docker.
+
+```bash
+~/ez init
+```
+
+or just create the docker stack based on an existing project
+
+```bash
+git clone yourproject.git application; cd application; ~/ez create
+```
+
+You will find more details on the [eZ Launchpad documentation](https://ezsystems.github.io/launchpad/).
+
+At this time you will have a working eZ Platform application with many services including Varnish, Solr, Redis etc.
+
+### Platform.sh integration
+
+To generate the key files for Platform.sh (`.platform.app.yaml` and `.platform`) just run:
+
+```bash
+~/ez platformsh:setup
+```
+
+eZ Launchpad will generate the files for you and you are then totally free to finetune them.
+
+> Note that SolR is not currently managed. You will have to set it up manually for Platform.sh.
+
+#### Environment variables (optional)
+
+One advantage among others with eZ Launchpad(thanks to Docker) and Platform.sh is you can easily reproduce the 
+Platform.sh environment variables in the `provisioning/dev/docker-compose.yml` file. It allows you to test locally
+the Platform.sh context so you stay consistent everywhere. 
```

Comment:
This sentence feels very clumsy.  Try:

"eZ Launchpad allows you to define environment variables in the `provisioning/dev/docker-compose.yml` file.  You may use that to set Platform.sh variables [link that to the variables page] to match Platform.sh environments so that you can keep your environment behavior in sync."

An example here would also be good.

Reply:
done it.